### PR TITLE
Refactor PayPal to use our backend 

### PR DESCRIPTION
**Pull request by meaku**
# Definition of Done

- [ ] The code does at least what is defined in the ticket and does not affect any other functionality
- [x] The PR has a meaningful title
- [ ] There is a link to the issue-id
- [x] The PR has been reviewed
- [ ] There are no 'hacks' or shortcuts refactoring is prefered

There are still open issues. But we deal with them in a seperate story #420

### CODE REVIEW

**Comment by dreampulse** [comment]
@meaku is this PR ready to review? Or is it still work in progress?

**Comment by meaku** [comment]
@henry-all3dp continued working on it and we changed the routes. Will update the frontend accordingly and we should be ready for a review next Wednesday.

**Comment by meaku** [comment]
It works against https://github.com/all3dp/printing-engine/pull/364. 

## Next steps are:

- [ ] fix tests
- [ ] add more tests 
- [ ] split order creation and payment call

File: config/development.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import defaultConfig from './default'
 
 export default defaultsDeep(
   {
-    printingEngineBaseUrl: 'http://localhost:8000/v1',
+    printingEngineBaseUrl: 'http://localhost:8000/v2',
```
Comment (meaku) [inline config/development.js]: I bumped the version for all routes as discussed.

File: src/app/service/paypal.js [hunk 2]
```diff
@@ -1,26 +1,51 @@
...
+          country_code: countryCode,
+          postal_code: zipCode,
+          state: stateCode
+        }
       }
     }
   ]
-  return paypal.rest.payment.create(env, client, {transactions})
+
+  return requestJson(PAYMENT_ENDPOINT, {method: 'POST', body: {orderId: offerId, transactions}})
```
Comment (meaku) [inline src/app/service/paypal.js]: `{ orderId: offerId }` is actually wrong. After splitting the order creation and the payment process, we will have an `orderId` to be sent along when calling the `payment` creation endpoint. 

Right now it's not really an issue as our API-side check does fail silently telling us that there is no such `oderId`. In order to release without splitting things apart, we could change the API and make `orderId` optional for now.

Comment (meaku) [reply src/app/service/paypal.js]: `UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Not found: Order not found`

Comment (meaku) [reply src/app/service/paypal.js]: As discussed with @henry-all3dp we can't make `orderId` optional. That means we have to split it up in the frontend before releasing.

**Comment by dreampulse** [comment]
Achtung! Es gab ein paar Änderungen im Playment prozess (invoice payment).

**Comment by meaku** [comment]
@dreampulse: After merging there was signature mismatch with `createOrder` and the way we create invoices. After talking to @henry-all3dp, we decided to remove the invoice function for now. Depending on Mikkels thoughts we should fix it to work with the newly deployed backend.

**Pull request merged**